pipboard: add show cases to pipboard (#52)
alibaba/pipcook: README.md

```sh
$ mkdir pipcook-example && cd pipcook-example
$ pipcook init
```

> ### Show Cases

Reviewer: How about "Playground" or "Pipboard"?

Author: I guess playground is more attractive. I would use 'Playground' :)

> If you are wondering what you can do in pipcook and where you can check your training logs and models, you could run pipboard first.

Reviewer: Add a blank line please :)

> If you are wondering what you can do in pipcook and where you can check your training logs and models, you could run pipboard first.

Reviewer: ```suggestion
If you are wondering what you can do in [Pipcook][] and where you can check training logs and models, you could start from Pipboard.
```

```sh
$ pipcook board
```

> you will see a web page prompt in your browser. Now you can have a look at MNIST showcase on the home page and play around there. If you want to train a model to recognize MNIST handwritten digits by yourself, you could try the example below.

Reviewer: ```suggestion
You will see a web page prompt in your browser, and there is a MNIST showcase on the home page and play around there. If you want to train a model to recognize MNIST handwritten digits by yourself, you could try the examples below.
```

> ### Examples

Reviewer: It should be removed for now, we have mentioned the examples at the previous section.

Reviewer: @utkobe Hey, the examples section should be included the playground? One is to remove the title, or downgrade the title level to "#### Examples".

Author: Yes, that makes sense. I just modified it

In this reposiory, We have prepared several examples, some of they are:

- [pipeline-mnist-image-classification](example/pipeline-example/pipeline-mnist-image-classification.js): pipeline for classific Mnist image classification problem.
- [pipeline-databinding-image-classification](example/pipeline-example/pipeline-databinding-image-classification.js): pipeline example to train the iamge classification task which is 
  to classifify [imgcook](https://www.imgcook.com/) databinding pictures.
- [pipeline-object-detection](example/pipeline-example/pipeline-object-detection.js): pipeline example to train object detection task which is for component recognition 
  used by imgcook.

> - [python-keras](example/python-nodejs-example/python-keras.js): example to use Python Keras library to train deep leraning network in js syntax and runtime.

Reviewer: See the bottom of the source: https://raw.githubusercontent.com/alibaba/pipcook/aab7e5d4a7d97166991826ee9966e33806d230e0/README.md, I have defined these links there, is this easier to read?

See [here](./example) for complete list, and it's easy and quick to run these examples. For example, to do a MNIST 
image classification, just run the following to start the pipeline:

```sh

> $ node examples/pipeline-example/pipeline-mnist-image-classification.js

Reviewer: How about renaming this to examples/pipeline/pipeline-mnist-image-classification?

## Community

#### IRC

> <img width="200" src="./community_qrcode.png">

Reviewer: An empty line is suggested :)

> Download DingTalk (an all-in-one free communication and collaboration platform) here: [English](https://www.dingtalk.com/static/en/download) | [中文](https://page.dingtalk.com/wow/dingtalk/act/download)

#### Who's using it

> <a href="https://www.imgcook.com"><img height="40" src="https://img.alicdn.com/tfs/TB1lle4yQzoK1RjSZFlXXai4VXa-200-64.png"></a>

Reviewer: Blank lines :)
